JPS_Ontology/ontology/ontotimeseries/OntoTimeSeries.csv

```diff
@@ -19,21 +19,20 @@ http://www.w3.org/2006/time#unitType ,Object Property,,,http://www.w3.org/2006/t
 hasForecastingModel,Object Property,,,Forecast,ForecastingModel,,A relationship between a forecast and its prediction model,https://www.theworldavatar.com/kg/ontotimeseries/,has forecasting model
 http://www.w3.org/2000/01/rdf-schema#label,Data Property,,,ForecastingModel,String,,Name or tag of a forecasting model,https://www.theworldavatar.com/kg/ontotimeseries/,
-hasURL,Data Property,,,ForecastingModel,String,,URL of storage location of a forecasting model,https://www.theworldavatar.com/kg/ontotimeseries/,has URL
+hasURL,Data Property,,,ForecastingModel,String,,URL of storage location of a (pre-trained) forecasting model,https://www.theworldavatar.com/kg/ontotimeseries/,has URL
 hasCovariate,Object Property,,,ForecastingModel,http://www.w3.org/2002/07/owl#Thing,,A relationship between a forecasting model for a particular quantity and associated covariates ,https://www.theworldavatar.com/kg/ontotimeseries/,has covariate
 hasTrainingTimeSeries,Object Property,,,ForecastingModel,TimeSeries,,A relationship between a forecasting model for a particular quantity and the time series used in deriving/training this model,https://www.theworldavatar.com/kg/ontotimeseries/,has training time series
 http://www.w3.org/2006/time#hasEnd,Object Property,,,http://www.w3.org/2006/time#Interval,http://www.w3.org/2006/time#Instant,,End of a temporal entity,http://www.w3.org/2006/time#,has end
 http://www.w3.org/2006/time#hasBeginning,Object Property,,,http://www.w3.org/2006/time#Interval,http://www.w3.org/2006/time#Instant,,Beginning of a temporal entity,http://www.w3.org/2006/time#,has beginning
 http://www.w3.org/2006/time#inTimePosition,Object Property,,,http://www.w3.org/2006/time#Instant,http://www.w3.org/2006/time#TimePosition,,"Position of an instant, expressed as a temporal coordinate or nominal value",http://www.w3.org/2006/time#,in time position
 http://www.w3.org/2006/time#TimePosition,Class,http://www.w3.org/2006/time#TemporalPosition,IS-A,,,,"A temporal position described using either a (nominal) value from an ordinal reference system, or a (numeric) value in a temporal coordinate system",http://www.w3.org/2006/time#,Time position
 http://www.w3.org/2006/time#TRS,Class,,,,,,"A temporal reference system, such as a temporal coordinate system (with an origin, direction, and scale), a calendar-clock combination, or a (possibly hierarchical) ordinal system",http://www.w3.org/2006/time#,Temporal coordinate system
-http://www.w3.org/2006/time#numericPosition,Data Property,,,http://www.w3.org/2006/time#TimePosition,Integer,,The (numeric) value indicating position within a temporal coordinate system,http://www.w3.org/2006/time#,Numeric position
+http://www.w3.org/2006/time#numericPosition,Data Property,,,http://www.w3.org/2006/time#TimePosition,decimal,,The (numeric) value indicating position within a temporal coordinate system,http://www.w3.org/2006/time#,Numeric position
```

Comment:
What is the motivation/need for this change?

Reply:
I believe the previous range  with `xsd:integer` is not compliant with the original definition by the time ontology (https://www.w3.org/TR/owl-time/#time:numericPosition). So, just wanted to fix this

Reply:
Ah, ok, I can see now that this is correct.

But this brings me to the next issue: Why are we repeating so many definitions from the time ontology here?

Reply:
While I can see these practical advantages, they are rather short-sighted. Redundant repeats of other people's definitions must be avoided as a matter of principle. Imagine what happens if/when these people decide to change their definitions - assuming they are eternally static isn't realistic. Sure, you might argue that if they change their definitions everything breaks anyway, but I'd rather not waste my time trying to find all the repeats... Things should be defined in one place, and in one place only. It's the same thing in software engineering: codes with lots of repetition are unmaintainable...

Reply:
Though we indicate the source of a concept or property in the defined-by annotation property, copying a definition completely from another ontology should be avoided as it might raise some issues. One can easily follow the defined-by link to check the definition. I would define a reused concept or property if I had to modify it to make it more specific or generic. I would leave the corresponding column empty if there is nothing new to communicate via the definition.

Reply:
Addressed by commit [47df945](https://github.com/cambridge-cares/TheWorldAvatar/pull/827/commits/47df9450fa6fc273a6c9e8d62787deffeaebda49)